packages/site/src/components/Connect.tsx

```diff
@@ -0,0 +1,78 @@
+import { ChangeEvent, FormEvent, FunctionComponent, useState } from 'react';
```

Comment:
Unrelated to this file, but do we not need the fonts anymore?
(couldn't comment on the deletion of fonts)

Reply:
Bootstrap uses native OS fonts, so I don't think it's necessary to use a different one.